policyengine_canada/parameters/gov/provinces/nb/tax/income/credits/age_amount/reduction_rate.yaml

```diff
@@ -0,0 +1,18 @@
+description: New Brunswick reduces the age amount tax credit at this rate.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: New Brunswick age amount credit reduction rate.
+  reference:
+  - title: 2023 New Brunswick Personal Tax Credit Return
+    href: https://www.unb.ca/hr/_assets/documents/td1nb-e.pdf 
```

Comment:
This seems like the better source, specifies 15% directly: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1nb-ws/td1nb-ws-23e.pdf